pkg/sdn/api/validation/validation.go

```diff
@@ -14,6 +15,10 @@ import (
 func ValidateClusterNetwork(clusterNet *sdnapi.ClusterNetwork) field.ErrorList {
 	allErrs := validation.ValidateObjectMeta(&clusterNet.ObjectMeta, false, oapi.MinimalNameRequirements, field.NewPath("metadata"))
 
+	if clusterNet.Name != sdnapi.ClusterNetworkDefault {
```

Comment:
Again, you can't break backwards compatibility with APIs.  Just because SDN
is the primary consumer in product, it does not mean that we can change how
the APIs work.  We can change some of the underlying behavior if necessary,
but it requires careful backwards support and proper communication with
users.

On Mon, Jun 13, 2016 at 3:18 PM, Ravi Sankar Penta <notifications@github.com
> wrote:

> In pkg/sdn/api/validation/validation.go
> <https://github.com/openshift/origin/pull/9147#discussion_r66850530>:
>
> > @@ -14,6 +15,10 @@ import (
> >  func ValidateClusterNetwork(clusterNet *sdnapi.ClusterNetwork) field.ErrorList {
> >  	allErrs := validation.ValidateObjectMeta(&clusterNet.ObjectMeta, false, oapi.MinimalNameRequirements, field.NewPath("metadata"))
> >
> > +	if clusterNet.Name != sdnapi.ClusterNetworkDefault {
>
> OpenShift SDN solution only looks at 'default' cluster network. What's the
> use of allowing arbitrary cluster network?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/9147/files/277121b53bdb5f9273ba0f69c2e604cb6967a82e..8f287cba588dc25bce3a57f2ae5fc4b097029df5#r66850530>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABG_p_r-qAduUeJs6P4z-KSLzTF35cEAks5qLa0IgaJpZM4Is66G>
> .
>